src/hostname/hostnamed.c

```diff
@@ -1754,6 +1930,12 @@ static const sd_bus_vtable hostname_vtable[] = {
                                 method_describe,
                                 SD_BUS_VTABLE_UNPRIVILEGED),
 
+        SD_BUS_METHOD_WITH_ARGS("SetAniColor",
+                                SD_BUS_ARGS("s", color, "b", interactive),
```

Comment:
please drop the "interactive" parameter.

A long long time ago the dbus protocol didn't carry a bit for this thing in its message header. that changed more than a decade ago, but this dbus api is older hence some of the api functions still carry the bit explicitly as a function parameter.

new code really shouldn't have that as a function parameter anymore.